pkg/common-controller/groupsnapshot_controller.go

```diff
@@ -0,0 +1,11 @@
+package common_controller
```

Comment:
Maybe we should rename groupsnapshot_controller.go to something like groupsnapshot_controller_helper.go as lots of the control logic is still in snapshot_controller.go and we actually only have one common controller?

Reply:
I don't mind that but currently most of the common login is in `snapshot_controller_base.go` and `snapshot_controller.go` has some specific functionality for snapshots. So I thought it made sense to have a similar `groupsnapshot_controller.go`.. What do you think?

Reply:
It was being discussed during KEP review that we should have just one controller to avoid having too many controllers. There will also be a problem of version skews if we add a new controller.

Reply:
Moved to `groupsnapshot_controller_helper.go`